Lesson_4/Tusk_1-3.py

```diff
@@ -0,0 +1,10 @@
+text = input('please input something: ')
+if text.isdigit():
+  if int(text) == 0:
+    print ('You entered a number and it is zero')
```

Comment:
As far as I know, `zero` is an even number, so you can skip a separate check for zero. You can fix it, or live as is since it is not a topic of our lesson